src/gmail.d.ts

```diff
@@ -223,6 +223,10 @@ interface GmailGet {
      */
     current_page(): GmailPageType;
     /**
+       Returns the number of emails shown per page
+     */
+    emails_per_page(): number;
+    /**
        Returns an object representation of the opened email contents
```

Comment:
Just pointing out that I really appreciate you updating the typescript references. That's great! :)